docs/SpaceTemplates.rst

```diff
@@ -111,13 +112,18 @@ In which:
 +-------------------------+------------------------------+----------------------------------------------------------------------------------------------+
 | **permissions**         | ``string``                   | The groups that have permissions to view and use this space template.                        |
 +-------------------------+------------------------------+----------------------------------------------------------------------------------------------+
+| **invitees**            | ``string``                   | The users and/or spaces that will be invited during space creation.                          |
++-------------------------+------------------------------+----------------------------------------------------------------------------------------------+
 | **homePageApplication** | ``SpaceApplication``         | The application to use in the home page of a space.                                          |
 +-------------------------+------------------------------+----------------------------------------------------------------------------------------------+
 | **applications**        | list of ``SpaceApplication`` | The list of the applications that are installed by default to a new space.                   |
 +-------------------------+------------------------------+----------------------------------------------------------------------------------------------+
 
 .. tip:: The ``permissions`` parameter is a list of membership expressions separated by ";" character.
-         Example: ``exo.social.space.template.community.permissions=*:/platform/administrators;*:/platform/web-contributors`` , only members of /platform/administrators and /platform/web-contributors can view ``community`` template and use it.
+         Example: ``exo.social.space.template.custom.permissions=*:/platform/administrators;*:/platform/web-contributors`` , only members of /platform/administrators and /platform/web-contributors can view ``custom`` template and use it.
+.. tip:: The ``invitees`` parameter is a list of user and space names separated by "," character.
+         Example: ``exo.social.space.template.custom.invitees=userA,space A,userB`` , the users userA, userB and all space A members will be added to the invitees for space having ``custom`` template.
+         The invitees will be added to the users input of the space creation popup. The user can remove any entry using the "x" icon.
```

Comment:
This does not seem to belong to dev guide. 
May be mentioned in user guide if not already @hghorbel

Reply:
Okay, I remove it